api/v1beta1/types_class.go

```diff
@@ -48,6 +48,7 @@ type AzureClusterClassSpec struct {
 	// - GermanCloud: "AzureGermanCloud"
 	// - PublicCloud: "AzurePublicCloud"
 	// - USGovernmentCloud: "AzureUSGovernmentCloud"
+	// - StackCloud: "HybridEnvironment"
```

Comment:
Is Azure Stack officially supported? Autorest is a deprecated library. I don't see Azure Stack listed in the new Cloud library - https://github.com/Azure/azure-sdk-for-go/blob/d165f8083de58e12135bce2102205081eaade930/sdk/azcore/cloud/cloud.go#L10.

Reply:
For example, the GermanCloud isn't supported anymore.

Reply:
I know there is an issue too to try and get rid of the autorest library implementations in CAPZ - https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2974

Reply:
No idea whether it is still supported, although I'm pretty sure it is still sold.